script/create-dist.py

```diff
@@ -146,22 +147,20 @@ def copy_chrome_binary(binary):
   os.chmod(dest, os.stat(dest).st_mode | stat.S_IEXEC)
 
 def copy_vcruntime_binaries():
-  with _winreg.OpenKey(_winreg.HKEY_LOCAL_MACHINE,
-                       r"SOFTWARE\Microsoft\VisualStudio\14.0\Setup\VC", 0,
-                       _winreg.KEY_READ | _winreg.KEY_WOW64_32KEY) as key:
-    crt_dir = _winreg.QueryValueEx(key, "ProductDir")[0]
+  vs_location = get_vs_location('[15.0,16.0)')
```

Comment:
`'[15.0,16.0)'` is a magic string, same as @MarshallOfSound's comment about `'14.13.26020'`

Could be read in from somewhere or, easier to do, make it a symbolic constant

Reply:
@ckerr see https://github.com/electron/electron/blob/master/script/lib/env_util.py#L66

I am simply using the logic that was introduced in https://github.com/electron/electron/pull/12537.

We do need to be version specific in this case, because we are explicitly targeting Visual Studio 2017